src/shared/lib/resources/counter.ts

```diff
@@ -0,0 +1,5 @@
+import { BodyWithErrors } from 'shared/lib/types';
+
+export interface UpdateValidationErrors extends BodyWithErrors {
+  name?: string[];
+}
```

Comment:
Can you add `export type UpdateRequestBody = null;` for completeness? Front-end will use it.